<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-A8BL1MV" failures="2" tests="2" name="Test(failed)" time="0.063" errors="0" timestamp="11 Nov 2020 17:41:43 GMT">
  <testcase classname="org.rerun.TestCase" name="method4" time="0.018">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.rerun.TestCase.method4(TestCase.java:22)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- method4 -->
  <testcase classname="org.rerun.TestCase" name="method5" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.rerun.TestCase.method5(TestCase.java:27)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- method5 -->
  <testcase classname="org.rerun.TestCase" name="method3" time="0.0">
    <ignored/>
  </testcase> <!-- method3 -->
  <testcase classname="org.rerun.TestCase" name="method1" time="0.0">
    <ignored/>
  </testcase> <!-- method1 -->
  <testcase classname="org.rerun.TestCase" name="method2" time="0.0">
    <ignored/>
  </testcase> <!-- method2 -->
</testsuite> <!-- Test(failed) -->
